Command/SendEmailCommand.php

```diff
@@ -39,7 +40,7 @@ protected function configure()
             ->setHelp(<<<EOF
 The <info>swiftmailer:spool:send</info> command sends all emails from the spool.
 
-<info>php %command.full_name% --message-limit=10 --time-limit=10 --recover-timeout=900 --mailer=default</info>
+<info>php app/console swiftmailer:spool:send --message-limit=10 --time-limit=10 --recover-timeout=900 --mailer=default</info>
```

Comment:
-1 for that btw. Using the placeholders is better, as it avoids having to maintain the command name in multiple places. Thus, the console file is not in ``app`` anymore in the Symfony 3 structure, meaning that your updated help message is wrong on Symfony 3.